### DISCUSSION TITLE
Do I have to maintain the two files in the settings or can I delete/move them to another folder?

### DISCUSSION THREAD

**Discussion by darkcnight** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#316] [Q&A]
As title.

**Comment by im-coder-lg** [comment]
You can, but you will have to login again to GAUDC

**Comment by Melvin-Abraham** [comment]
If you choose to move the **Key File** and **Saved Token File**, you will have to update the corresponding path in the Assistant Settings for it to work as intended.
